src/MessageHandler/ActivityPub/Inbox/CreateHandler.php

```diff
@@ -95,4 +97,14 @@ private function handlePage(): void
             }
         }
     }
+
+    private function handlePrivateMessage(): void
+    {
+        $this->messageManager->createMessage($this->object);
+    }
+
+    private function handlePrivateMentions(): void
+    {
+        // TODO implement private mentions
```

Comment:
TODO left?

Reply:
Not for this PR, but for later, yes

Reply:
Sure OK. I understand. I just wanted to point it out, you never know. right?